src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -78,14 +80,15 @@ export interface TableProps {
   items: SavedObjectWithMetadata[];
   itemId: string | (() => string);
   totalItemCount: number;
-  onQueryChange: (query: any, filterFields: string[]) => void;
+  onQueryChange: (query: any, filterFields?: string[]) => void;
```

Comment:
Seems unrelated changes.

Reply:
Have deleted it according to your suggestion.